src/acceleration/MVQNAcceleration.cpp

```diff
@@ -120,9 +120,10 @@ void MVQNAcceleration::computeUnderrelaxationSecondaryData(
     Eigen::VectorXd &values = data->values();
     values *= _initialRelaxation; // new * omg
     Eigen::VectorXd &secResiduals = _secondaryResiduals[id];
-    secResiduals                  = data->oldValues.col(0); // old
-    secResiduals *= 1.0 - _initialRelaxation;               // (1-omg) * old
-    values += secResiduals;                                 // (1-omg) * old + new * omg
+    const auto       oldValues    = data->previousIteration();
+    secResiduals                  = oldValues; // old
```

Comment:
```suggestion
    secResiduals                  = data->previousIteration();
```
Unnecessarily complicated.